### DISCUSSION TITLE
Add Specflow.Assist.Dynamic port for Reqnroll

### DISCUSSION THREAD

**Discussion by marcosnsantana** [reqnroll/Reqnroll#350] [Ideas]
Team, 

I have successfully created a port of https://github.com/marcusoftnet/SpecFlow.Assist.Dynamic for Reqnroll, which allows dynamic instances and dynamic sets to be generated on Reqnroll tables. I have been able to successfully test this functionality in my local project migrated to Reqnroll by making this port work for dynamic table generation (as my project is 100% based on dynamic Specflow table data generation).

Can I please receive permissions to submit a Pull Request for this? The feature branch is named feature/ReqnrollAssistDynamic, but I'm currently not authorized to push this branch. Git output:

`Pushing feature/ReqnrollAssistDynamic
Remote: Permission to reqnroll/Reqnroll.git denied to marcosnsantana.
Failed to push to the remote repository. See the Output window for more details.
Error encountered while pushing to the remote repository: Git failed with a fatal error.
unable to access 'https://github.com/reqnroll/Reqnroll/': The requested URL returned error: 403
`

**Comment by gasparnagy** [comment]
We are looking forward to it!

For security reasons pushing new branches to the Reqnroll repos are not enabled by default, unless someone have a proven record of a successful PR.

I know it is a bit of extra work, but please make a fork of the Reqnroll project into your own GitHub account apply the changes to that one (to a new branch) and send a PR from that branch. (See https://github.com/reqnroll/Reqnroll/blob/main/CONTRIBUTING.md for details.)

**Comment by RajeshPoola** [comment]
@gasparnagy I ported files from [SpecFlow.Assist.Dynamic](https://github.com/marcusoftnet/SpecFlow.Assist.Dynamic) into ReqnRoll project and raised PR https://github.com/reqnroll/Reqnroll/pull/397. Could someone please have a look and approve the MR or provide some feedback. Thanks.

**Reply by olegKoshmeliuk** [reply]
We had  opened PR for same feature around 2 weeks ago. https://github.com/reqnroll/Reqnroll/pull/377
Actually I ported this lib 3 weeks ago based on your question https://github.com/reqnroll/Reqnroll/issues/371#issuecomment-2561985579

**Reply by RajeshPoola** [reply]
> We had opened PR for same feature around 2 weeks ago. #377 Actually I ported this lib 3 weeks ago based on your question [#371 (comment)](https://github.com/reqnroll/Reqnroll/issues/371#issuecomment-2561985579)

Thanks very much. I will close my PR
